tests/foreman/cli/test_repository.py

```diff
@@ -1849,14 +1850,13 @@ def test_negative_restricted_user_cv_add_repository(self):
             {'organization-id': org['id']})
         self.assertEqual(len(repos), 0)
 
-    @skip_if_bug_open('bugzilla', 1378442)
-    @tier1
-    def test_positive_upload_content_srpm(self):
-        """Create repository and upload a SRPM content
+    @tier2
+    def test_positive_upload_remove_srpm_content(self):
+        """Create repository , upload and remove a SRPM content
```

Comment:
Create repository, upload and remove an SRPM content

Reply:
Disagree due to [1][2]
[1] https://wiki.centos.org/HowTos/RebuildSRPM
[2] https://www.theclassroom.com/use-vs-sentence-2221937.html